EMIP/1001-2000/EMIP01969.xml

```diff
@@ -57,6 +65,80 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         </extent>
                      </supportDesc>
                   </objectDesc>
+                  <handDesc>
+                  <handNote xml:id="h1" script="Ethiopic">
+                  <seg type="rubrication">
+                  <foreign xml:lang="gez">ኢግዚአብሔር፡</foreign>
+                  <desc>The word God is rubricated in the first three works. Occasionally God is written in alternating red and black letters (e.g., <locus target="#2v"/>)</desc>
+                  <foreign xml:lang="gez">ማርያም፡</foreign>
+                  <desc>The word Mary is rubricated in the final two works. </desc>
+                  </seg>
+                  <desc>Columetric layout of text: <locus target="#110v #111r"/> (Ps. 150); <locus target="#119v #119r #120v #120r"/> (tenth Biblical Canticle)</desc>
+                  </handNote>
+                  </handDesc>
+                  <decoDesc>
+                  <decoNote type="miniature" xml:id="d01"><locus target="#148v"/><desc> of a rider on a horse</desc></decoNote>
+                  <decoNote type="frame" xml:id="d02">
+                  Decorative designs:
+                  <locus target="#1r #13v #20r #40r #48v #72r #105r"/> (haräg)
```

Comment:
I would not use "decorative design" for "harag", I would use "ornamental band" instead (crude, elaborate, monochrome, bicolour, polychrome).